### ISSUE TITLE
Space in the Bug Report issue form for providing a list of enabled addons

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#5438]
### Why this would be helpful/solve a problem

Sometimes knowing what addons somebody is using can be a huge help in diagnosing issues [(example)](https://github.com/ScratchAddons/ScratchAddons/issues/5185#issuecomment-1275439034).

### What this change does

In the [Bug Report issue form](https://github.com/ScratchAddons/ScratchAddons/blob/master/.github/ISSUE_TEMPLATE/---bug.yml), adds `Enabled Addons:` to the Environment field value. You can add the list of addons you've enabled next to it.

### Possible alternatives

Have a section below Environment dedicated to listing enabled addons.

### Demonstration(s) and additional context

- [ ] A Scratch Addons feature to copy a list of your enabled addons would make this easier, so we should probably work on that first: https://github.com/ScratchAddons/ScratchAddons/issues/5068#issue-1358845776

**Comment by penguinmoose** [comment]
Do you think this input field for enabled addons should be required? It can be tedious to list all the addons you have enabled.

**Comment by BroJac5246** [comment]
> Do you think this input field for enabled addons should be required? It can be tedious to list all the addons you have enabled.

No, as they're some cases where it won't be necessary, such as bug reports from Discord or feedback.

**Comment by DNin01** [comment]
> Do you think this input field for enabled addons should be required? It can be tedious to list all the addons you have enabled.

Not all bug reports will need addon lists, so no, but it would be great if everyone were able to provide a list of enabled addons if necessary or if requested.

Also, we'd do this after SA gets a feature to list enabled addons, making providing enabled addons much easier.

**Comment by WorldLanguages** [comment]
I think we could add a new button inside "more settings" that would open an URL such as `https://scratchaddons.github.io/list-of-enabled-addons/?number=14ki1cr8xydnb283jqd4&version=v1.28.0` in a new tab, and then the user could paste that URL into the issue form.
We could also try (in the future) to include the value of settings, and not just whether addons are enabled or not, into the URL.

We should also add a button to the list-of-enabled-addons website that lets you import the list as a settings JSON file (and at the moment, it would use default settings, since those aren't stored in the URL)
